assets/model_monitoring/components/src/shared_utilities/df_utils.py

```diff
@@ -13,6 +13,8 @@ def get_numerical_columns(column_dtype_map: dict) -> list:
         column
         for column in column_dtype_map
         if column_dtype_map[column] in ["float", "double", "decimal"]
+        or column_dtype_map[column] == "int"
+        and len(set(column))/len(column) >= 0.05
```

Comment:
Should this be < 0.05 to be numerical? Sorry I am not familiar with this logic. From this code, I see we are checking unique character rate in the column name < 5% to be both categorical and numerical.

Reply:
https://ml.azure.com/experiments/id/dd803194-3afd-45b9-aa3e-44b4d5a0e3a7/runs/hungry_wing_mvw836dcc1?wsid=/subscriptions/4f26493f-21d2-4726-92ea-1ddd550b1d27/resourcegroups/model-monitor-demo/providers/Microsoft.MachineLearningServices/workspaces/model-monitor-demo-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

Reply:
Also, with the heuristic here it should be < 5% I believe for numerical.